pkg/store/tree.go

```diff
@@ -160,3 +151,45 @@ func addPathToTree(path, value string, nodeif *interface{}) error {
 
 	return nil
 }
+
+func handleLeafValue(nodemap map[string]interface{}, value *change.TypedValue, pathelems []string, floatAsStr bool) {
+	switch value.Type {
+	case change.ValueTypeEMPTY:
+		// NOOP
+	case change.ValueTypeSTRING:
+		(nodemap)[pathelems[1]] = (*change.TypedString)(value).String()
+	case change.ValueTypeINT:
+		(nodemap)[pathelems[1]] = (*change.TypedInt64)(value).Int()
+	case change.ValueTypeUINT:
+		(nodemap)[pathelems[1]] = (*change.TypedUint64)(value).Uint()
+	case change.ValueTypeDECIMAL:
+		(nodemap)[pathelems[1]] = (*change.TypedDecimal64)(value).Float()
+	case change.ValueTypeFLOAT:
+		if floatAsStr {
```

Comment:
maybe I'm missing the point but why is this needed ? not supported in gnmi ?

Reply:
not supported in YGOT - doesn't seem to want float as raw (in JSON) - wants it as string - whatever